prompts/2024/english/prompt_2024_01/AnaColon_summary_Bingham_and_Ratcliff_2024

```diff
@@ -0,0 +1,17 @@
+Complex multicellularity has evolved five separate times in eukaryotes but has never evolved in prokaryotes. Complex multicellularity has been defined as a large organism composed of many cells that lives for an extensive period of time. Some prokaryotes are multicellular but they do not have the morphologies of complex multicellular organisms. 
+
+This paper proposes a new hypothesis as to why prokaryotes lack complex multicellularity compared to eukaryotes. When bottle neck events causing a reduced efficient population size cells are divided into groups. In response to this eukaryotes will undergo genomic expansion while prokaryotes will undergo genomic deletion. These differences in responses could be an explanation as to why eukaryotes have been able to evolve complex multicellularity and prokaryotes have not. 
+
+Eukaryotic have transposons and introns that facilitate genomic expansion through replication. Prokaryotes tend to have deletion bias where they will delete parts of their genome. These subcellular mechanisms influence their response to bottleneck effects. 
```

Comment:
'through replication' is a little vague - do you get the nuances of this? These 'mobile genetic elements' are kind of like viruses, and get replicated and moved around in our genomes within the lifespan of single cells and without reproduction of the organism itself. And importantly, they often take other, nearby pieces of the genome with them - so a transposon that happens to be next to an unrelated gene will get copied to a different part of the genome, and maybe the second copy of the transposon will have a fragment of that other gene next to it. This allows pieces of genes to get mixed around and recombined, as well as duplicated.